### DISCUSSION TITLE
Is there a way to disable the magnet functionality for a specific Link?

### DISCUSSION THREAD

**Discussion by tkaabbc** [clientIO/joint#2710] [Q&A]
### Introduction

Hi team, 
I use a `Link` to measure length, when the Link gets close to other elements, it gets snapped by the `magnet`, as shown by the yellow border in the image. 

Is there a way to **disable the magnet functionality for this specific Link**, so it doesn't snap to any elements? 

(I only found a way to disable the magnet on elements, but that's not what I want because other Links still need to snap to the elements normally.)
<img width="246" alt="image" src="https://github.com/user-attachments/assets/2e119508-6ac0-41e1-ac79-3540f3abb95f">

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment]
You can use [validateConnection()](https://docs.jointjs.com/api/dia/Paper/#validateconnection) paper option callback and return `false` if `linkView` parameter is of a certain type.
